stdpopsim/catalog/ensembl_info.py

```diff
@@ -1,2 +1,2 @@
 # File autogenerated from Ensembl REST API. Do not edit.
-release = 100
+release = 101
```

Comment:
Can you confirm that the script ran for all genomes? We don't want to have some at release 100 and others at 101 (although I'm sure there's probably no changes).

Reply:
> Can you confirm that the script ran for all genomes?

Yep.